tests/inference/test_query_http_requests.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
+import pytest
+
+
+def script_with_args(model_name, streaming_response, max_new_tokens, temperature, top_p):
+    config_path = "../.github/workflows/config/" + model_name + ".yaml"
```

Comment:
suggest to get current script absolute path as base path and set other path relative to it, then this script will be independent from where your run from.